packages/playground/src/components/account/account-edit/account-edit.tsx

```diff
@@ -65,7 +65,7 @@ export class AccountEdit {
           />
           <form-button
             class="button"
-            onButtonPressed={e => this.formSubmitionHandler()}
+            onButtonPressed={e => this.formSubmissionHandler()}
```

Comment:
I think this `this.formSubmissionHandler.bind(this)` works

Reply:
No idea :) I just fixed the spelling mistakes.
@patience-tema-baron What do you say?